pandas/core/generic.py

```diff
@@ -7115,12 +7115,61 @@ def _tz_localize(ax, tz, ambiguous):
     # Numeric Methods
     def abs(self):
         """
-        Return an object with absolute value taken--only applicable to objects
-        that are all numeric.
+        Return a Series/DataFrame with absolute numeric value of each object.
```

Comment:
Can you repalce "object" with "element"? That's our usual noun for items in a frame.